modules/farm_fd2/src/entrypoints/direct_seeding/direct_seeding.submission.e2e.cy.js

```diff
@@ -81,7 +118,43 @@ describe('Direct Seeding: Submission tests', () => {
      * lib.submitForm function that is called when the "Submit"
```

Comment:
Be sure to edit/delete/move/clean up comments so that they agree with the way your code works now.